server/services/caseloadService.ts

```diff
@@ -128,7 +129,11 @@ export default class CaseloadService {
         )
 
         // Create a case in the list in status NOT_STARTED
-        return { ...offender, licenceStatus: LicenceStatus.NOT_STARTED, licenceType } as CaseTypeAndStatus
+        return {
+          ...offender,
+          licenceStatus: prisonInRollout(offender?.prisonId) ? LicenceStatus.NOT_STARTED : LicenceStatus.NOT_IN_PILOT,
+          licenceType,
+        } as CaseTypeAndStatus
       })
```

Comment:
Where a case is not started but in a prison that is outside the pilot areas, shows as a status `Outside pilot`